README.md

```diff
@@ -4,48 +4,25 @@
 [![codecov](https://img.shields.io/codecov/c/github/triplea-game/triplea/master.svg?style=flat-square)](https://codecov.io/gh/triplea-game/triplea)
 [![TripleA license](https://img.shields.io/github/license/triplea-game/triplea.svg?style=flat-square)](https://github.com/triplea-game/triplea/blob/master/LICENSE)
 
+
+##  [TripleA Home Page](http://triplea-game.org/)
+
 TripleA is an open source gaming community, free to play, 100% open source and volunteer run
 
+ 
 ## [Download and install TripleA](http://triplea-game.org/download/)
-
 ![screenshot from 2018-02-08 22-58-33](https://user-images.githubusercontent.com/12397753/36015523-a4e28a24-0d23-11e8-84c0-c4bd0ee19ce0.png)
 
-- Online lobby, find and join live games
-- Play by email
-- Play vs AI
-- Many community created maps available for in-game download, you can create your own with your own rule set!
 
-## Contact Us
-- [Bug report](https://github.com/triplea-game/triplea/issues/new)
-- [Feature requests](https://forums.triplea-game.org/category/42/feature-requests-and-ideas)
+## Contact Us:
+- [Github Issues (Bug Reports)](https://github.com/triplea-game/triplea/issues/new)
 - [Gitter](https://gitter.im/triplea-game/social)
 
-## Up-Time Reports
+## Is it Running? Availability Dashboards
 - [Lobby and Forums](https://stats.uptimerobot.com/14RYqsN5m)
 - [Game Hosting Bots](https://stats.uptimerobot.com/2WVgrCzO4)
 
 ## Documentation Links
-- Developer, getting started: https://github.com/triplea-game/triplea/tree/master/docs/dev
-- Moderator, how-to: https://forums.triplea-game.org/ (secret admin section)
-- Setting up Linode servers: https://github.com/triplea-game/lobby#adding-linode-servers
-- Lobby+Bot Ops (install, check status, restart): https://github.com/triplea-game/lobby
-- Forums Ops: https://github.com/triplea-game/triplea/tree/master/docs/admin#nodebb-forums
-- Dice Server (TODO): https://github.com/triplea-game/triplea/tree/master/docs/admin#the-dice-server
-
-## License
```

Comment:
Removing this heading will require an update to the license notice URL:

https://github.com/triplea-game/triplea/blob/f43cfdd8d71e9e73d9e76f90c334925b8deac1f0/game-core/src/main/java/games/strategy/triplea/UrlConstants.java#L45

Reply:
Which also means we'll break existing clients  :face_with_head_bandage: 

Good catch. 

The README.md is/was too heavy. @ssoloff beyond what was done here, can you suggest alternatives or more things that can be done to cleanly tuck 'license' away to another page and reduce redundant material?